docs/SnapshotTesting.md

```diff
@@ -39,7 +39,7 @@ exports[`renders correctly 1`] = `
 `;
 ```
 
-The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process. Jest uses [pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format) to make snapshots human-readable during code review. On subsequent test runs, Jest will compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the test runner found a bug in your code (in this case, it's `<Link>` component) that should be fixed, or the implementation has changed and the snapshot needs to be updated.
+The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process. Jest uses [pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format) to make snapshots human-readable during code review. On subsequent test runs, Jest will compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the test runner found a bug in your code (in this case, its `<Link>` component) that should be fixed, or the implementation has changed and the snapshot needs to be updated.
```

Comment:
Is this change intended?

Reply:
yea, this is more correct grammatically, otherwise we say "in this case, it is `<Link>` component". We might have meant to convey some other meaning, though?

Reply:
Maybe avoid using "it" altogether?
```suggestion
The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process. Jest uses [pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format) to make snapshots human-readable during code review. On subsequent test runs, Jest will compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the test runner found a bug in your code (`<Link>` component in this case) that should be fixed, or the implementation has changed and the snapshot needs to be updated.
```

Reply:
[`c6d0e51` (#10691)](https://github.com/facebook/jest/pull/10691/commits/c6d0e512ab1e321830449310bb02ca81176e8399) and [`e1c214d` (#10691)](https://github.com/facebook/jest/pull/10691/commits/e1c214d6a72709219e42c5dd950d607d2af8f441)